client/signup/steps/theme-selection/index.jsx

```diff
@@ -66,13 +66,34 @@ class ThemeSelectionStep extends Component {
 	};
 
 	renderThemesList() {
+		const pressableWrapperClassName = classNames( {
+			'theme-selection__pressable-wrapper': true,
```

Comment:
```
633 classNames
573 classnames
```
Take your pick :) I think `classNames`, since this is also the way it is named in the official package documentation.